app/App/Account/index.js

```diff
@@ -147,10 +147,18 @@ class _Footer extends React.Component {
                   style={{ pointerEvents: this.state.allowInput ? 'auto' : 'none' }}
                   onClick={() => {
                     if (this.state.allowInput) {
+                      const { address, symbol, decimals, logoURI, name, chainId } = req.token
                       link.send('tray:resolveRequest', req, null)
                       link.send('tray:action', 'navDash', {
-                        view: 'notify',
-                        data: { notify: 'addToken', notifyData: { token: req.token } }
+                        view: 'tokens',
+                        data: {
+                          notify: 'addToken',
+                          notifyData: {
+                            tokenData: { address, symbol, decimals, logoURI, name },
```

Comment:
what was the reason for changing this interface? seems like previously the notify data was just the token, which is pretty clean and makes sense for an add token dialog. now we have "tokenData" and also chainId and address which are separated out. also address appears twice